src/colorizer/utils/data_utils.ts

```diff
@@ -19,6 +18,7 @@ import {
   ThresholdType,
   TrackPathColorMode,
 } from "src/colorizer/types";
+import type { RenderCanvasStateParams } from "src/colorizer/viewport/IRenderCanvas";
```

Comment:
I plan to move this type out of `viewport/IRenderCanvas` and into `viewport/types` in a follow-up PR.